docs/cli/user-manual.md

```diff
@@ -210,4 +211,30 @@ Test and validate current application. This command will validate your applicati
 ***
 
 # `teamsfx publish`
-Publish the app to Teams.
\ No newline at end of file
+Publish the app to Teams.
```

Comment:
Any additional notes, e.g.,
- The log folder
- SPFX not supported
...

Reply:
Good idea, added.